lib/Try/Tiny.pm

```diff
@@ -29,6 +29,8 @@ BEGIN {
   *_HAS_SUBNAME = ($su || $sn) ? sub(){1} : sub(){0};
 }
 
+our @_finally_guards;
```

Comment:
`our @some_variable` is generally perceived as an invite of *please feel free to fondle me if you think it is justified*. Even (or often especially) if rhe variable is `_` prefixed.

In the case of `T::T` there is no scenario where such "fondling" is justified, therefore a `my %stackstate` with `local()` on a key is a better solution.